test/create-webpack-app/test.utils.js

```diff
@@ -0,0 +1,426 @@
+/* eslint-disable node/no-unpublished-require */
```

Comment:
This file is not needed, it's already present in the `test-utils` directory.

Reply:
changed and reduced redundancy ([ea92e56](https://github.com/webpack/webpack-cli/pull/4167/commits/ea92e56708b98755fb12fd670f1f3eccb929bfd1)) significantly without affecting other suites.
checked locally by testing all but 1 test fails.